### ISSUE TITLE
Option to Unmount/Remove label

### ISSUE THREAD

**Issue by shorobor** [google/samba-documents-provider#56]
No way to unmount obsolete shares from Downloads app's sidebar. Please add option to track added shares and option to remove or edit/reconfigure.

**Comment by Veltys** [comment]
Yes, please; it's impossible to do that without help.

**Comment by lukegackle** [comment]
Yes please, currently the only way to do that is to clear data for the Android samba client which Clears everything...

**Comment by uKER** [comment]
+1
We need a list of current shares, with options to edit or delete them.
Currently, any change in the share's or the server's name, or any change in the login or password means being stuck with an useless share.

**Comment by BenMcLean** [comment]
This app just isn't usable without this.

**Comment by OdinGitDat** [comment]
Does there not exist any workaround for this?

**Comment by uKER** [comment]
Not from the UI at least.
Maybe using filesystem access and edit configuration files you can change the configs, but I don't think that's the point.

**Comment by OdinGitDat** [comment]
I know that's not the point, but I'm still looking for a workaround, maybe using Termux? I'm not very skilled with android command line.
